etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/database/PostgresDaoTest.groovy

```diff
@@ -137,4 +137,59 @@ class PostgresDaoTest extends Specification {
         then:
         thrown(SQLException)
     }
+
+    def "fetchMetadata happy path with hasValidData:true works"() {
+        given:
+        def mockDriver = Mock(SqlDriverManager)
+        def mockPreparedStatement = Mock(PreparedStatement)
+        def mockResultSet = Mock(ResultSet)
+        def expected = null
+        Connection MockConn = Mock(Connection)
```

Comment:
Minor detail, but is there any reason `MockConn` name starts with uppercase? Same for the other test

Reply:
Good catch, done.